### PR TITLE
wgpuDeviceGetLimits need to add return

### PR DESCRIPTION
**Pull request by xdestiny110**
We should add return statement otherwise it will always return false. @kainino0x

### CODE REVIEW

File: src/library_webgpu.js [hunk 1]
```diff
@@ -771,6 +771,7 @@ var LibraryWebGPU = {
     setLimitValueU32('maxComputeWorkgroupSizeY', {{{ C_STRUCTS.WGPULimits.maxComputeWorkgroupSizeY }}});
     setLimitValueU32('maxComputeWorkgroupSizeZ', {{{ C_STRUCTS.WGPULimits.maxComputeWorkgroupSizeZ }}});
     setLimitValueU32('maxComputeWorkgroupsPerDimension', {{{ C_STRUCTS.WGPULimits.maxComputeWorkgroupsPerDimension }}});
+    return true;
```
[the remark this answers is not in the document]
Comment (xdestiny110) [reply src/library_webgpu.js]: Done in [8e50f8a](https://github.com/emscripten-core/emscripten/pull/20808/commits/8e50f8ab87e8dc458ecef264373995f372fec9ec)

**Review by kainino0x — APPROVED** [review]
LGTM!

File: src/library_webgpu.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (sbc100) [reply src/library_webgpu.js]: Yeah I think you are correct.  I'm not sure which I prefer TBH.   `1` has the nice property that its 3 bytes smaller :)

**Comment by floooh** [comment]
Linking to a related ticket I wrote a while ago: https://github.com/emscripten-core/emscripten/issues/20278

**Pull request merged**